freqtrade/optimize/backtesting.py

```diff
@@ -738,7 +738,7 @@ def get_valid_price_and_stake(
         if order_type == 'limit':
             new_rate = strategy_safe_wrapper(self.strategy.custom_entry_price,
                                              default_retval=propose_rate)(
-                pair=pair, current_time=current_time,
+                pair=pair, trade=trade, current_time=current_time,
```

Comment:
@xmatthias I don't get why 'LocalTrade' type cause an isse for 'custom_entry_price', and it is working fine for 'custom_exit_price', the only difference is see there is that it is 'Optional'. What do you think?

Reply:
LocalTrade != Trade ... 

it's a typing problem, really - but we don't want to use LocalTrade directly in the strategy (also not as types) - as that has other downsides.
wrap line - add type ignore (other backtesting.py callbacks will have the same.

Reply:
I will do that now, Thank you